app/src/server/fineTuningProviders/openpipe/trainingConfig.ts

```diff
@@ -32,10 +33,10 @@ const baseConfig: Partial<AxolotlConfig> = {
   optimizer: "adamw_bnb_8bit",
   output_dir: "__placeholder__",
   dataset_processes: 8,
-  val_set_size: 0.05,
   logging_steps: 1,
   save_safetensors: true,
-  eval_steps: 0.1,
+  val_set_size: 0,
+  // eval_steps: 0.1,
```

Comment:
Do we need this comment?

Reply:
We might want to turn evals back on -- I just turned it off as a temporary fix for a bug that will likely be fixed in Axolotl.